examples/tts/magpietts.py

```diff
@@ -12,10 +12,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import lightning.pytorch as pl
+from pytorch_lightning import Trainer
```

Comment:
hmm..this seems odd since https://github.com/blisc/NeMo/pull/47 has fixed it and this line attempts to revert the change. 

Could you please rebase the PR on top of HEAD of blisc:magpietts_2503?
```
git remote add blisc git@github.com:blisc/NeMo.git
git fetch blisc/magpietts_2503
git rebase blisc/magpietts_2503
```